grpc/pom.xml

```diff
@@ -24,17 +24,17 @@
         <dependency>
             <groupId>io.dropwizard</groupId>
             <artifactId>dropwizard-core</artifactId>
-            <version>2.0.8</version>
+            <version>2.1.6</version>
```

Comment:
After I got the code compiling + the `import` step working correctly, I got a bunch of dropwizard-related errors trying to spin up a server locally. I messed around for a bit and found that updating these few dropwizard-related libs fixed the issue, so 🤷